packages/nextjs/test/config.test.ts

```diff
@@ -389,7 +390,21 @@ describe('Sentry webpack plugin config', () => {
       ]);
     });
 
-    it('has the correct value when building serverful server bundles', async () => {
+    it('has the correct value when building serverful server bundles using webpack 4', async () => {
+      const finalWebpackConfig = await materializeFinalWebpackConfig({
+        userNextConfig,
+        incomingWebpackConfig: serverWebpackConfig,
+        incomingWebpackBuildContext: { ...serverBuildContext, webpack: { version: '4' } },
```

Comment:
Just so it's a little more realistic, 

```suggestion
        incomingWebpackBuildContext: { ...serverBuildContext, webpack: { version: '4.28.3' } },
```

Reply:
Done